code/src/components/headerfooter/Footer.styles.js

```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components';
+
+export const FooterStyles = styled.footer`
+    margin-top: 15px;
+    padding: 15px;
+    background-color: rgb(245, 245, 245, .5);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  
+  p {
+    text-align: center;
+    font-family: 'Yeseva One', cursive;
+    color: #4e54c8;
+    margin: 0;
+  }
+
+  a {
+    color: #4e54c8;
+  }
+
+  a:visited {
+    color: #4e54c8
+  }
+
+  img, .contact-logo {
+    width: 30px;
+    height: 30px;
+    object-fit: cover;
+    opacity: .75;
+    color: #4e54c8;
```

Comment:
Nice that you changed the colour of the icons to match the background! I hadn’t realised you could change the colour of Font Awesome icons!